src/components/cart_product_card/CartProductCard.jsx

```diff
@@ -0,0 +1,39 @@
+import "./CartProductCard.css";
+const CartProductCard = () => {
+	return (
+		<div class="card horizontal-card">
+			<img
+				class="card-img-container horizontal-card-image "
+				src="https://assets.myntassets.com/f_webp,dpr_1.5,q_60,w_210,c_limit,fl_progressive/assets/images/14971104/2021/7/29/e894ad8a-2c03-44ff-98ac-f0e9226ddc7f1627553869161AfroJackMenGreen1.jpg"
+				alt="shoes"
+			/>
+			<div class="card-body">
+				<h2 class="card-heading">Running shoes</h2>
+				<h3 class="card-subheading">by Nike</h3>
+				<p class="pv-1">
+					<span class="typo-label">Rs. 1499 </span>
+					<span class="typo-subtext text-line-through">Rs 5999</span>
```

Comment:
Here you have use span element for price of product. According to mdn :
The <span> element is a generic inline container for phrasing content, which does not inherently represent anything. It can be used to group elements for styling purposes (using the class or id attributes).